### ISSUE TITLE
Some silkscreen symbols are not being rendered completely

### ISSUE THREAD

**Issue by mtiutiu** [wokwi/easyeda2kicad#15]
**Description:**
Seems that some symbols present on the PCB silkscreen (top and/or bottom) are not converted completely leading to an incomplete drawing. See the "**high voltage symbols**" in the attached pictures.

**Expected result:**
Any silkscreen symbols should be fully translated.

Original:
![Screenshot from 2020-01-29 15-34-47](https://user-images.githubusercontent.com/2891463/73428804-c6c62b80-4342-11ea-9955-de2c128cc59d.png)

Conversion result:
![Screenshot from 2020-01-30 09-30-27](https://user-images.githubusercontent.com/2891463/73429027-4d7b0880-4343-11ea-9923-8a9c875963dd.png)

Project files:
[project.zip](https://github.com/wokwi/easyeda2kicad/files/4132711/project.zip)

KiCAD version (latest stable): **5.1.5**
easyeda2kicad version: **1.3.4**

If any other details are needed please let me know.
Thanks.

**Comment by urish** [comment]
Can you please check again with latest version from master?

**Comment by mtiutiu** [comment]
@urish 

Hi, the conversion process is crashing with an exception now:
```
/home/mtiutiu/Tmp/easyeda2kicad/dist/board.js:319
    var isSmd = shapes.some(function (shape) { return shape[0] === 'pad' && shape[2] === 'smd'; });
                                                           ^

TypeError: Cannot read property '0' of null
    at /home/mtiutiu/Tmp/easyeda2kicad/dist/board.js:319:60
    at Array.some (<anonymous>)
    at convertLib (/home/mtiutiu/Tmp/easyeda2kicad/dist/board.js:319:24)
    at convertShape (/home/mtiutiu/Tmp/easyeda2kicad/dist/board.js:391:21)
    at /home/mtiutiu/Tmp/easyeda2kicad/dist/board.js:403:148
    at Array.map (<anonymous>)
    at Object.convertBoard (/home/mtiutiu/Tmp/easyeda2kicad/dist/board.js:403:118)
    at Object.<anonymous> (/home/mtiutiu/Tmp/easyeda2kicad/dist/main.js:17:84)
```

**Comment by urish** [comment]
Should be fixed by #20, can you give it another go?

**Comment by mtiutiu** [comment]
Yep. Works now as expected. See attached picture.
![Screenshot from 2020-02-04 10-57-32](https://user-images.githubusercontent.com/2891463/73729215-4dad4680-473d-11ea-8a45-865a28632056.png)

**Comment by urish** [comment]
Awesome! released as 1.4.0
